porch/func/Dockerfile

```diff
@@ -33,6 +33,8 @@ COPY porch/api/go.mod porch/api/go.sum porch/api/
 
 RUN cd porch ; go build -v google.golang.org/grpc k8s.io/klog/v2
 
+COPY pkg pkg
```

Comment:
This is a side effect of https://github.com/GoogleContainerTools/kpt/pull/2961.
This path is rooted at `kpt` instead of `kpt/porch`.
We need https://github.com/GoogleContainerTools/kpt/blob/d6e4a118a265365d605befeb582c4cc59e361569/pkg/fn/eval.go#L37-L43